### ISSUE TITLE
w08d04 HW

### ISSUE THREAD

**Issue by Karen-O** [ga-dc/wdidc5-project3#6]
https://github.com/Karen-O/Project-3

**Comment by RobertAKARobin** [comment]
Even if you haven't decided on an idea, please pick one and create user stories. It's helpful for us, and more importantly, for you!

**Comment by adambray** [comment]
@Karen-O 

Can you please update your repo to include your User Stories and ERDs?

**Comment by Karen-O** [comment]
What if they keep changing?

I know it's somewhat backwards to do it this way, but can I wait till the
end to submit "nice" versions? (And in the meantime, maybe just upload
rough drafts, assuming they're readable-- see attached?)

For instance, I didn't want to "officially" submit last night's version
because by the time I finished, I'd already crossed out/ changed half, and
still making alterations even now. (The real thing will be typed, of
course.)

As for the ERD, I've completely reworked it (such that I switched back to
pen/paper), and so I'll basically be remaking my original Lucidpress
version from scratch. But again, since it needs to "match" the project, I'm
hesitant to say it's finalized.

Sorry I didn't respond earlier; barely looked at my phone today!
